Source/Resources/Content/MSGraph/DeviceManagement/DeviceEnrollmentConfigurations/deviceEnrollmentWindowsHelloForBusinessConfiguration - All users and all devices.json

```diff
@@ -7,7 +7,7 @@
       }
     }
   ],
-  "description": "This is the default Windows Hello for Business configuration applied with the lowest priority to all users regardless of group membership.",
+  "description": "What is it: The default Windows Hello for Business configuration applied with the lowest priority to all users regardless of group membership. \n Simeon Notes: Simeon's baseline disables Hello For Business including the option to set a PIN on enrollment. Simeon's experience has found that the majority of business do not want this option turned on. \n Learn More: https://docs.microsoft.com/en-us/mem/intune/protect/identity-protection-windows-settings",
```

Comment:
**What does this do?**
Configures Windows Hello for Business settings. The baseline disables Windows Hello for Business.

**Why should you use this?**
Simeon has found that the majority of businesses do not want the Windows Hello for Business functionality turned on.

**What is the end-user impact?**
Users will not be prompted to set up Windows Hello when they log in to a device for the first time.

**Learn more**
[Windows Hello for Business](https://docs.microsoft.com/en-us/mem/intune/protect/identity-protection-windows-settings)